values/jobs/autoconfig.gotmpl

```diff
@@ -0,0 +1,36 @@
+{{- $v := .Values }}
+{{- $c := $v.charts }}
+{{- $k := $c.keycloak }}
+type: Job
+name: autoconfig
+
+image:
+  repository: docker.io/otomi/otomi-e2e
+  tag: autoconfig
+  pullPolicy: Always
+
+script: | 
+    {{ if eq $v.otomi.hasExternalIDP false }}
+    echo "Creating Keycloak user"
+    yarn cypress run --record false --env BASE_URL={{$v.cluster.domainSuffix}} --spec 'cypress/integration/otomi-applications/10-keycloak.feature'
+    {{ end }}
+    {{ if eq $c.gitea.enabled true }}
```

Comment:
Please move this condition under `{{ if eq $v.otomi.hasExternalIDP false }}` block, because it is not possible to configure automatically Drone if there is externalIDP.